.travis.yml

```diff
@@ -5,22 +5,27 @@ rvm:
   - 2.1.10
```

Comment:
Man, I remember the day when I was super excited for Ruby 2.1, but nowadays I think we can even remove this from Travis as well. 2.2 is quite compatible and gives that restriction from the `?.` operator.